tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+        local = {
+        source  = "hashicorp/local"
+        version = "2.4.0"
+        }
+        kubectl = {
+        source  = "gavinbunney/kubectl"
+        version = "1.14.0"
+        }
+    }
+    required_version = ">= 1.0.0"
+    }
+    ```
+
+2. Add the Scaleway provider and its configuration. Refer to the Scaleway Terraform provider documentation for more information on how to [configure the provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs).
+    ```
+    provider "scaleway" {
+    region  = "fr-par"
+    }
+    ```
+
+3. Now create a dedicated VPC and a Private Network where we will deploy the Kapsule cluster. VPC and Private Networks are region-wide, so you can use them to host a cluster spread on multiple availability zones.
+    ```
+    resource "scaleway_vpc" "vpc_multi_az" {
+    name = "vpc-multi-az"
+    tags = ["multi-az"]
+    }
+
+    resource "scaleway_vpc_private_network" "pn_multi_az" {
+    name   = "pn-multi-az"
+    vpc_id = scaleway_vpc.vpc_multi_az.id
+    tags   = ["multi-az"]
+    }
+    ```
+
+4. Add the Kapsule cluster and the three pools, each in a different availability zone but in the same Private Network, to effectively make the cluster multi-AZ.
+Refer to the Scaleway Terraform provider documentation for available [Kapsule parameter]([https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_cluster), there is no specific configuration for the multi-AZ mode on the `scaleway_k8s_cluster`.
+For the [pools configuration](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/k8s_pool) it's the `zone` parameter that allows placing each one in a different availability zone.
+    ```
+    resource "scaleway_k8s_cluster" "kapsule_multi_az" {
+    name        = "kapsule-multi-az"
+    tags        = ["multi-az"]
+
+    type    = "kapsule"
+    version = "1.28"
+    cni     = "cilium"
+
+    delete_additional_resources = true
+
+    autoscaler_config {
+        ignore_daemonsets_utilization = true
+        balance_similar_node_groups   = true
+    }
+
+    auto_upgrade {
+        enable                        = true
+        maintenance_window_day        = "sunday"
+        maintenance_window_start_hour = 2
+    }
+
+    private_network_id = scaleway_vpc_private_network.pn_multi_az.id
+    }
+
+    resource "scaleway_k8s_pool" "pool-multi-az" {
+    for_each = {
+        "fr-par-1" = 1,
+        "fr-par-2" = 2,
+        "fr-par-3" = 3
+    }
+
+    name                   = "pool-${each.key}"
+    zone                   = each.key
+    tags                   = ["multi-az"]
+    cluster_id             = scaleway_k8s_cluster.kapsule_multi_az.id
+    node_type              = "PRO2-XXS"
+    size                   = 2
+    min_size               = 2
+    max_size               = 3
+    autoscaling            = true
+    autohealing            = true
+    container_runtime      = "containerd"
+    root_volume_size_in_gb = 32
+    }
```

Comment:
Fixed indent & added node type in the for_each to ease the use case when the node type is not available in all zones

```suggestion
    resource "scaleway_k8s_cluster" "kapsule_multi_az" {
        name        = "kapsule-multi-az"
        tags        = ["multi-az"]

        type    = "kapsule"
        version = "1.28"
        cni     = "cilium"

        delete_additional_resources = true

        autoscaler_config {
            ignore_daemonsets_utilization = true
            balance_similar_node_groups   = true
        }

        auto_upgrade {
            enable                        = true
            maintenance_window_day        = "sunday"
            maintenance_window_start_hour = 2
        }

        private_network_id = scaleway_vpc_private_network.pn_multi_az.id
    }

    resource "scaleway_k8s_pool" "pool-multi-az" {
        for_each = {
            "fr-par-1" = "PRO2-XXS",
            "fr-par-2" = "PRO2-XXS",
            "fr-par-3" = "PRO2-XXS"
        }

        name                   = "pool-${each.key}"
        zone                   = each.key
        tags                   = ["multi-az"]
        cluster_id             = scaleway_k8s_cluster.kapsule_multi_az.id
        node_type              = "${each.value}"
        size                   = 2
        min_size               = 2
        max_size               = 3
        autoscaling            = true
        autohealing            = true
        container_runtime      = "containerd"
        root_volume_size_in_gb = 32
    }
```